CONTRIBUTING.md

```diff
@@ -61,4 +63,13 @@ format C code. You can run this with:
 
 ```console
 $ clang-format -i *.c *.h
-```
\ No newline at end of file
+```
+
+## Go code linting
+
+`go-geos` uses [`golangci-lint`](https://golangci-lint.run/) to lint Go code.
+You can run it with:
+
+```
+$ golangci-lint run .
```

Comment:
The `.` should be removed. If it is present then it tells `golangci-lint` to only check the current package, whereas `golangci-lint` should check every package in the module.

Reply:
Good point! Fixed